src/pyskindose/phantom_class.py

```diff
@@ -2,13 +2,26 @@
 import numpy as np
 import os
 import pandas as pd
-import plotly.graph_objs as go
-import plotly.offline as ply
+import plotly.graph_objects as go
 from stl import mesh
 from typing import Dict
```

Comment:
Reorder and restructure imports.

3 sections (each in alphabetical order)
- standard library
- 3rd party packages
- this packages modules

Reply:
done